upstream-community-operators/aws-service/aws-service.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,178 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: aws-service-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    categories: "OpenShift Optional, Cloud Provider"
```

Comment:
Should we use the `OpenShift Optional` category for upstream?

Reply:
This is a great point at Sami. @robszumski ?

Reply:
Good point @SamiSousa, what are your thoughts @robszumski?

Reply:
Spoke with @robszumski and was told not to include the `OpenShift Optional` category.

Reply:
Resolved in the latest commit.